pkg/connectors/datacatalog/clients/datacatalog_grpc.go

```diff
@@ -211,3 +214,9 @@ func ConvertDataCatalogGrpcRespToOpenAPIResp(result *pb.CatalogDatasetInfo) (*da
 func (m *grpcDataCatalog) CreateAsset(in *datacatalog.CreateAssetRequest, creds string) (*datacatalog.CreateAssetResponse, error) {
 	return &datacatalog.CreateAssetResponse{AssetID: "testAssetID"}, nil
 }
+
+// just adding this dummy implementation as we are going to remove grpc support soon.
+// Then this file will be removed. Till then we provide a dummy implementation.
+func (m *grpcDataCatalog) DeleteAsset(in *datacatalog.DeleteAssetRequest, creds string) (*datacatalog.DeleteAssetResponse, error) {
```

Comment:
There is no such functionality in the proto, so why is this needed?

Reply:
We need to do this as we are enforcing the data catalog interface on the grpc implementation (https://github.com/fybrik/fybrik/blob/451c4192920c132350d83b1678edbff3f34c90a7/pkg/connectors/datacatalog/clients/datacatalog_grpc.go#L250).